ontology/rml-core.owl

```diff
@@ -0,0 +1,517 @@
+         rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+         rdfs:label "subject" .
+
+
+###  http://w3id.org/rml/subjectMap
+:subjectMap rdf:type owl:ObjectProperty ;
+            rdfs:domain :TriplesMap ;
+            rdfs:range :SubjectMap ;
+            rdfs:comment "A SubjectMap element to generate a subject from a logical source record."@en ;
+            rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+            rdfs:label "subject map" .
+
+
+###  http://w3id.org/rml/termType
+:termType rdf:type owl:ObjectProperty ;
+          rdfs:domain :TermMap ;
+          rdfs:range [ rdf:type owl:Class ;
+                       owl:unionOf ( :BlankNode
+                                     :IRI
+                                     :Literal
+                                   )
+                     ] ;
+          rdfs:comment "A string indicating whether subject or object generated using the value from column name specified for rr:column should be an IRI reference, blank node, or a literal."@en ;
+          rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+          rdfs:label "term type" .
+
+
+#################################################################
+#    Data properties
+#################################################################
+
+###  http://w3id.org/rml/child
+:child rdf:type owl:DatatypeProperty ;
+       rdfs:domain :Join ;
+       rdfs:range xsd:string ;
+       rdfs:comment "Names a column in the child table of a join."@en ;
+       rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+       rdfs:label "child" .
+
+
+###  http://w3id.org/rml/language
+:language rdf:type owl:DatatypeProperty ;
+          rdfs:domain :ObjectMap ;
+          rdfs:range xsd:string ;
+          rdfs:comment "Specified the language for the object component for the generated triple from a logical source record."@en ;
+          rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+          rdfs:label "language" .
+
+
+###  http://w3id.org/rml/parent
+:parent rdf:type owl:DatatypeProperty ;
+        rdfs:domain :Join ;
+        rdfs:range xsd:string ;
+        rdfs:comment "Names a column in the parent table of a join."@en ;
+        rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+        rdfs:label "parent" .
+
+
+###  http://w3id.org/rml/reference
+:reference rdf:type owl:DatatypeProperty ;
+           rdfs:comment "A valid expression that selects values from the source data. The reference should conform to the syntax rules/grammar of the specified Reference Formulation." ;
+           rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+           rdfs:label "reference"^^xsd:string .
+
+
+###  http://w3id.org/rml/template
+:template rdf:type owl:DatatypeProperty ;
+          rdfs:domain :ExpressionMap ;
+          rdfs:range xsd:string ;
+          rdfs:comment "A template (format string) to specify how to generate a value for a subject, predicate, or object, using one or more columns from a logical source record."@en ;
```

Comment:
"value for a subject, predicate, or object": that's restrictive, could be a GraphMap, LanguageMap etc.
Rephrase into "value for an expression map"